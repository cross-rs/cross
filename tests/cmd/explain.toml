bin.name = "cross"
args = ["--explain", "E0004"]
