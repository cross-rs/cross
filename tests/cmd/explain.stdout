This error indicates that the compiler cannot guarantee a matching pattern for
one or more possible inputs to a match expression. Guaranteed matches are
required in order to assign values to match expressions, or alternatively,
determine the flow of execution.
...